app/services/sla_breach_service.rb

```diff
@@ -0,0 +1,16 @@
+class SLABreachService
+  attr_accessor :report_generated_at
+  def initialize
+    @report_generated_at = Time.now.utc
+  end
+
+  def breach_threshold
+    3.business_days.before(@report_generated_at)
+  end
+
+  # attention_needed_since is present and is before breach threshold
+  # returns data in the format { vita_partner_id: breach_count }
+  def attention_needed_breach
+    Client.sla_tracked.group(:vita_partner_id).where(Client.arel_table[:attention_needed_since].lteq(breach_threshold)).count(:id)
```

Comment:
This took me a minute to wrap my brain around.. may want to refactor to make it more clear - not urgent or blocking though

Reply:
@sniemeyer13 do you think it would make it more readable if i broke it into separate lines so I could comment what each part is doing?